src/ucs/sys/event_set.c

```diff
@@ -167,3 +181,13 @@ void ucs_event_set_cleanup(ucs_sys_event_set_t *event_set)
     close(event_set->epfd);
     ucs_free(event_set);
 }
+
+ucs_status_t ucs_event_set_fd_get(ucs_sys_event_set_t *event_set, int *fd_p)
+{
+    if (event_set == NULL) {
+         return UCS_ERR_INVALID_PARAM;
+    }
+    *fd_p = event_set->epfd;
+
```

Comment:
this space line should be moved before line 190

Reply:
Changed below.

```C
     if (event_set == NULL) {
          return UCS_ERR_INVALID_PARAM;
     }

    *fd_p = event_set->epfd;
     return UCS_OK;
```